exercises/hamming/example.tt

```diff
@@ -17,11 +17,11 @@ class HammingTest < Minitest::Test<% test_cases.each do |test_case| %>
   # The version number refers to the version of the problem you solved,
   # not your solution.
   #
-  # Define a constant named VERSION inside of Hamming.
+  # Define a constant named VERSION inside of BookKeeping.
```

Comment:
Nothing has mentioned something called `BookKeeping` until this moment, and it might feel like it came out of nowhere (_book-keeping? What book-keeping? Am I supposed to know what that means?_).

Since this is the first (non-hello-world) exercise, I suggest that we be a little bit more verbose here. Maybe something like:

```
# Create a class or module named BookKeeping which defines a constant named VERSION.
# You can do this in the same file as the rest of your solution.
```